packages/augur-ui/src/modules/modal/components/modal-view.tsx

```diff
@@ -56,10 +56,10 @@ function selectModal(type, props, closeModal, modal) {
       return (
         <>
           {/* MOBILE */}
-          <ModalAddFunds />
+          <ModalAddFunds autoSelect={false} />
 
           {/* DESKTOP */}
-          <ModalAddFunds {...props} autoSelect />
+          <ModalAddFunds autoSelect={true} />
```

Comment:
if a prop is a boolean, you don't need to explicitly pass `someProp={true}` you can simply just put in `someProp` and that will count as true.